adapters/beachfront/beachfronttest/noop/six-nine-combo--response-order--banner-adm-nurl-adm--FAIL.json

```diff
@@ -0,0 +1,458 @@
+{
```

Comment:
I think the `noop` directory and the test files in it should be removed since they aren't working tests. You can always add them to the `supplemental` folder in another PR when you have them passing.

Reply:
Will do. The big thing that I want to get working in this is the ability for the bidder to send requests to beachfront's video endpoint as a single request with multiple imps. As it is it sends one request for each video imp, which is obviously a waste of resources. That is what I'm build up to. Before that, I'll push with the noop directory removed for this PR, then another PR with your suggested tests above.